rules/pkg-normalize.js

```diff
@@ -22,6 +22,14 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && typeof repository !== 'object') {
```

Comment:
I think we should add an extra check for `repository.url`?

```js
if ((typeof repository !== 'string' && typeof repository !== 'object') || repository.url === undefined) {
```

This way we can ignore the `&& repository.url` down below at line 29

Because currently if they set `"repository": {}`, it will call `{}.includes('github.com')` which will fail.